rebar.lock

```diff
@@ -13,6 +17,10 @@
  {<<"hamcrest">>,{pkg,<<"basho_hamcrest">>,<<"0.4.1">>},2},
  {<<"jsx">>,{pkg,<<"jsx">>,<<"2.10.0">>},0},
  {<<"lager">>,{pkg,<<"lager">>,<<"3.7.0">>},0},
+ {<<"paillier">>,
```

Comment:
The TravisCI build seems to be failing on building this dependency. Any ideas why? I didn't have time to look into it yet, but this is something that needs to be fixed before merging, whether it's the build environment or something in `paillier`.

Reply:
This was my mistake, it's fixed now.